frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableNode.tsx

```diff
@@ -140,26 +151,54 @@ export const TableNode: FC<Props> = ({
-                            (isHighlighted ||
-                              highlightedHandles?.includes(handleId)) &&
-                              styles.handleCardinalityHighlighted,
-                          ])}
-                        />
+                        <>
+                          <CardinalityZeroOrOneLeftIcon
+                            className={clsx([
+                              styles.handleCardinality,
+                              styles.handleCardinalityLeft,
+                              (isHighlighted ||
+                                highlightedHandles?.includes(handleId)) &&
+                                styles.handleCardinalityHighlighted,
+                            ])}
+                          />
+
+                          <span
+                            className={clsx([
+                              styles.handleCardinalityComment,
+                              styles.handleCardinalityCommentLeft,
+                              (isHighlighted ||
+                                highlightedHandles?.includes(handleId)) &&
+                                styles.handleCardinalityCommentHighlighted,
+                            ])}
+                          >
+                            1
+                          </span>
+                        </>
                       ))
                       .with('ONE_TO_MANY', () => (
-                        <CardinalityZeroOrManyLeftIcon
-                          className={clsx([
-                            styles.handleCardinality,
-                            styles.handleCardinalityLeft,
-                            (isHighlighted ||
-                              highlightedHandles?.includes(handleId)) &&
-                              styles.handleCardinalityHighlighted,
-                          ])}
-                        />
+                        <>
+                          <CardinalityZeroOrManyLeftIcon
+                            className={clsx([
+                              styles.handleCardinality,
+                              styles.handleCardinalityLeft,
+                              (isHighlighted ||
+                                highlightedHandles?.includes(handleId)) &&
+                                styles.handleCardinalityHighlighted,
+                            ])}
+                          />
+
+                          <span
+                            className={clsx([
+                              styles.handleCardinalityComment,
+                              styles.handleCardinalityCommentLeft,
+                              (isHighlighted ||
+                                highlightedHandles?.includes(handleId)) &&
+                                styles.handleCardinalityCommentHighlighted,
+                            ])}
+                          >
+                            n
+                          </span>
+                        </>
```

Comment:
💭 TableNode is getting quite complex and I would like to split the files little by little.